source/NVDAObjects/IAccessible/adobeAcrobat.py

```diff
@@ -137,15 +143,30 @@ def _getNodeMathMl(self, node):
 		yield "</%s>" % tag
 
 	def _get_mathMl(self):
+		if self.pdDomNode is None:
+			log.debugWarning("_get_mathMl: self.pdDomNode is None!")
+			raise LookupError
 		# There could be other stuff before the math element. Ug.
+		mathMl = self.pdDomNode.GetValue()
+		log.debug(
+			f'\n_get_mathMl: math recognized: {mathMl.startswith("<math")}, child count={self.pdDomNode.GetChildCount()}',
```

Comment:
Why are we querying the same thing multiple times here and below?

Reply:
I would also consider wrapping debug queries in `if log.isEnabledFor(log.DEBUG):` for performance